src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/RuntimeEnvironment.cs

```diff
@@ -22,8 +22,8 @@ public static bool FromGlobalAccessCache(System.Reflection.Assembly a)
             return false;
         }
         public static string GetRuntimeDirectory()
-        {
-            return Path.GetDirectoryName(typeof(object).Assembly.Location) + Path.DirectorySeparatorChar;
+        { 
```

Comment:
This is behavior change for .NET Core. Runtime directory (ie where CoreLib lives) is typically different from APPBASE directory (ie where your .exe lives).